parse-zoneinfo/src/line.rs

```diff
@@ -877,6 +859,190 @@ pub struct Rule<'a> {
     pub letters: Option<&'a str>,
 }
 
+impl<'a> Rule<'a> {
+    fn from_str(input: &'a str) -> Result<Self, Error> {
+        let mut state = RuleState::Start;
+        for part in input.split_ascii_whitespace() {
```

Comment:
This no longer parses a rule with a comment?

`zic.c` has a `getfields` method (line 3722) that returns when it encounters a comment sign `#`.
It also supports quotation marks `"` surrounding each field. Maybe something we should make an iterator that works similar instead of using `split_ascii_whitespace`?